unraid_mcp/tools/health.py

```
@@ -1,186 +1,264 @@
+            runtime = sub_status.get("runtime", {})
+            conn_state = runtime.get("connection_state", "unknown")
+            if conn_state in ("error", "auth_failed", "timeout", "max_retries_exceeded"):
+                diagnostic_info["summary"]["in_error_state"] += 1
+            if runtime.get("last_error"):
+                connection_issues.append({
+                    "subscription": sub_name,
+                    "state": conn_state,
+                    "error": runtime["last_error"],
+                })
+
+        return diagnostic_info
```

Ruff (0.15.0)

[warning] Consider moving this statement to an `else` block

(TRY300)